.envrc

```diff
@@ -47,9 +47,7 @@ fi
 if command -v nix &> /dev/null && [ -z ${DISABLE_NIX+x} ]
 then
     if nix flake metadata > /dev/null; then
-        if type nix_direnv_watch_file &> /dev/null; then
```

Comment:
No need to check anymore as `watch_file` is provided by direnv itself and not by nix-direnv